build.gradle.kts

```
@@ -6,7 +6,7 @@ plugins {
 
 group = "com.iridium"
 version = "4.0.2"
-description = "IridiumSkyblock"
+description = "PeachySkyblock"
```

You misspelled "dasSkyblock"
```suggestion
description = "dasSkyblock"
```